.travis.yml

```diff
@@ -0,0 +1,24 @@
+sudo: required
```

Comment:
That's unfortunate :/ I just gave it a shot as well. 

I'll get our Travis account ready for this on Monday.